pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/NonPersistentTopics.java

```diff
@@ -428,6 +430,8 @@ public void getList(
                 final List<String> nonPersistentTopics =
                         topics.stream()
                                 .filter(name -> !TopicName.get(name).isPersistent())
+                                .filter(topic -> includeSystemTopic
+                                        ? true : !pulsar().getBrokerService().isSystemTopic(topic))
```

Comment:
LGTM

Just a trivial comment, could we add a method in `PersistentTopicsBase` for this filter, it seems that it appears 3 times.

Reply:
ok. fixed.